lib/icinga/apiactions.cpp

```diff
@@ -813,6 +813,21 @@ Dictionary::Ptr ApiActions::ExecuteCommand(const ConfigObject::Ptr& object,
 	if (local) {
 		ClusterEvents::ExecuteCommandAPIHandler(origin, execParams);
 	} else {
+		/* Check if the child endpoints have Icinga version >= 2.13 */
+		Zone::Ptr localZone = Zone::GetLocalZone();
+		for (const Zone::Ptr& zone : ConfigType::GetObjectsByType<Zone>()) {
+			/* Fetch immediate child zone members */
+			if (zone->GetParent() == localZone) {
```

Comment:
Oh, and you could additionally check whether `zone` can access `endpointPtr->GetZone()`.

If it can't, it doesn't get the message from RelayMessage().